app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+class UsersController < ApplicationController
+  
+  def index
+  	@user = User.all
+  end
+
+  def show
+  	@user = User.find(params[:id])
+  end
+
+  def create
+    @user = User.create(user_params)
+
+    redirect_to users_path
+  end
+
+  def new
+  	@user = User.new
+  end
+
+  def edit
+  	@user = User.find(params[:id])
```

Comment:
move this code to the private method (def resource) and reuse it other actions(show/edit/destroy)